scripts/init-compliance-tool-for-environment.sh

```diff
@@ -1,57 +1,85 @@
 #!/usr/bin/env bash
 set -e
 
-if [ ! $# -eq 2 ] ; then
-	echo "this script requires the environment and a persistentId to be passed in as arguments. e.g."
-	echo "$ ./init-compliance-tool.sh <environment> <persistentId>"
+if [ ! $# -eq 3 ] ; then
+	echo "this script requires the environment and a persistentId to be passed in as arguments, along with either the '--matching' flag or the name of a file containing matching dataset json. e.g."
+	echo "$ ./init-compliance-tool.sh <environment> <persistentId> --matching"
```

Comment:
Should we have an additional line of example showing passing in a file, for completeness?

Reply:
Disregard this, I've started to look at the next PR and have seen the examples there.